src/transformers/models/timm_backbone/modeling_timm_backbone.py

```diff
@@ -108,10 +108,10 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
         return super()._from_config(config, **kwargs)
 
     def freeze_batch_norm_2d(self):
-        timm.layers.freeze_batch_norm_2d(self._backbone)
+        timm.utils.model.freeze_batch_norm_2d(self._backbone)
```

Comment:
@amyeroberts There is two changes which is not directly related to RTDETR.
current timm model has been changed!

Reply:
@SangbumChoi Just to make sure I've understood correctly - the change is coming from the timm side? If so, we'll need to have some version dependent logic, or pin the version of timm >= to this critical commit

Reply:
@amyeroberts I assume that you are right. I am not sure but i think it worked at the first place. I recommend to have dependent logic.

https://github.com/huggingface/pytorch-image-models/blob/6e6f3686a7e06bcba37bbd3b7c755f04a516a1e7/timm/utils/model.py#L130

Reply:
You're right - that commit was from 3 years ago. Let's just update here to be correct :)